charts/backstage/values.yaml

```diff
@@ -28,6 +28,12 @@ global:
       existingSecret: ""
       # -- Instead of generating a secret value, use fo;lowing value
       value: ""
+  # -- Global configuration values for PostgreSQL
+  postgresql:
+    # -- The authentication details of the Postgres database
+    auth:
+      # -- Name of existing secret to use for PostgreSQL credentials
+      existingSecret: ''
```

Comment:
Do we need to have this one defined as empty? Isn't this the default anyways?

Reply:
Without it, `helm template` fails with 
```
error calling include: template: backstage/templates/_helpers.tpl:31:23: executing "janus-idp.backstage.existingSecret" at <.Values.global.postgresql.auth.existingSecret>: nil pointer evaluating interface {}.auth
```
and the only way that I found to fix this error was to include `global.postgresql.auth.existingSecret`

Reply:
Interesting. I'm seeing the same. Which is surprising since the upstream helm chart should set this to `{}` instead of nil.

Anyways, we can solve this through a nullable `?.` pipeline alternative in Helm:


```diff
- {{- if .Values.global.postgresql.auth.existingSecret ...
+ {{- if ((((.Values).global).postgresql).auth).existingSecret ...
```